src/dataset-builder/DatasetBuilder.ts

```diff
@@ -460,33 +458,25 @@ interface RequestAccessModalProps {
 
 const RequestAccessModal = (props: RequestAccessModalProps) => {
   const { onDismiss, cohorts, conceptSets, valueSets, snapshotId } = props;
-  const [name, setName] = useState('');
-  const [researchPurposeStatement, setResearchPurposeStatement] = useState('');
-
-  const required = { presence: { allowEmpty: false } };
-  const errors = validate({ name, researchPurposeStatement }, { name: required, researchPurposeStatement: required });
-
-  const nameId = _.uniqueId('');
-  const researchPurposeId = _.uniqueId('');
 
   return h(
     Modal,
     {
-      title: 'Requesting access',
+      title: 'Access request created in Terra',
       showX: false,
       onDismiss,
+      width: 500,
+      cancelText: 'Return to AxIN Overview',
       okButton: h(
         ButtonPrimary,
         {
-          disabled: errors,
-          tooltip: errors && Utils.summarizeErrors(errors),
           onClick: withErrorReporting('Error creating dataset request')(async () => {
             await DataRepo()
```

Comment:
I believe this `onClick` is no longer correct - I believe this should be a `Link` now? (eventually to redcap?)

Reply:
I agree, should this be a future ticket, because we don't know what the link is?

Reply:
I think that we should move it to be a link, and just have it link to nowhere for now, and then a future ticket to put in the correct link. That way we get the behavior and styling down now.